### ISSUE TITLE
projects create with an invalid name should fail

### ISSUE THREAD

**Issue by plambert** [rundeck/rundeck-cli#180]
```
% rd projects create -p "invalid name"
# Created project: 
#     invalid name
% echo $?
0
% rd projects list
% 
```

This should fail, and ideally say why, and definitely not give a return code of 0.

**Comment by gschueler** [comment]
looks like this is a rundeck bug

**Comment by gschueler** [comment]
fyi; this should be fixed in 2.11.4+.  We should additionally validate the input on the `rd` side

**Comment by HowardHong** [comment]
The rd cli will also create a project with an invalid name.  
And it seems you can only delete it using rd.  Deleting the project from the GUI doesn't work.
